### PR TITLE
Add UI descriptions and asset metas to a bunch of assets

### PR DESCRIPTION
**Pull request by WeirdRubberDuck**
Add UI description and asset.meta to a bunch of asset files (mainly scale objects, night sky and examples)

To make them more useful for our users, especially with the improvements to the node meta panel from PR https://github.com/OpenSpace/OpenSpace-WebGuiFrontend/pull/154

### CODE REVIEW

File: data/assets/educational/scale/kuala_lumpur_tower.asset [hunk 1]
```diff
@@ -83,3 +84,15 @@ end)
 asset.export(ScaleModel)
 asset.export("UpdatePositionAction", UpdatePositionAction.Identifier)
 asset.export("ResetPositionAction", ResetPositionAction.Identifier)
+
+
+asset.meta = {
+  Name = "Scale Model - Kuala Lumpur Tower",
+  Version = "1.0",
+  Description = [[A 1:1 scale model of the Kuala Lumpur Tower. Per default it is placed
+    at its actual position in Kuala Lumpur. But the asset also includes actions to move
```
Comment (ylvaselling) [inline data/assets/educational/scale/kuala_lumpur_tower.asset:93]: Kuala Lumpur, Malaysia to be consistent with the previous assets:)

File: data/assets/examples/dashboarditems.asset [hunk 2]
```diff
@@ -100,3 +100,15 @@ asset.export(Mission)
 asset.export(PropertyValue)
 asset.export(ElapsedTime)
 asset.export(InputState)
+
+
+asset.meta = {
+  Name = "Dashboard Items Example",
+  Version = "1.0",
+  Description = [[Examples of different types of dashboard items.
+    These are dynamic information texts that wil be shown over the rendering
```
Comment (ylvaselling) [inline data/assets/examples/dashboarditems.asset:109]: * will

File: data/assets/examples/discs.asset [hunk 3]
```diff
@@ -51,3 +51,13 @@ end)
 
 asset.export(BasicDisc)
 asset.export(FullEllipticDisc)
+
+
+asset.meta = {
+  Name = "Example Discs",
+  Version = "1.0",
+  Description = [[Example of different types of rendered discs.]],
```
Comment (ylvaselling) [inline data/assets/examples/discs.asset:59]: Examples?

File: data/assets/examples/globerotation.asset [hunk 4]
```diff
@@ -55,3 +55,15 @@ asset.onDeinitialize(function()
...
 
 asset.export(ExampleGlobeRotation)
+
+
+asset.meta = {
+  Name = "Model Example",
+  Version = "1.0",
+  Description = [[An example that demonstrates how to load a 3D model from a geometry
+    file, together with some basic funcitonality. The model is placed on the surface
+    of a panet using a GlobeRotation and GlobeTranslation.]],
```
Comment (ylvaselling) [inline data/assets/examples/globerotation.asset:65]: planet

File: data/assets/examples/globetranslation.asset [hunk 5]
```diff
@@ -72,3 +72,15 @@ end)
 
 asset.export(ExampleFixedHeight)
 asset.export(ExampleAdaptiveHeight)
+
+
+asset.meta = {
+  Name = "GlobeTranslation Example",
+  Version = "1.0",
+  Description = [[An example that demonstrates how to place object on a planet surface
```
Comment (ylvaselling) [inline data/assets/examples/globetranslation.asset:80]: an object

File: data/assets/educational/scale/kuala_lumpur_tower.asset (hunk 1 above)
Comment (ylvaselling) [inline data/assets/educational/scale/kuala_lumpur_tower.asset:93]:
```suggestion
    at its actual position in Kuala Lumpur, Malaysia. But the asset also includes actions to move
```

File: data/assets/educational/scale/school_bus.asset [hunk 6]
```diff
@@ -41,6 +41,8 @@ local ScaleModel = {
   },
   GUI = {
     Name = "School bus",
+    Desription = [[The average length of a full size US school bus is above 35 feet
+      (10.7 meters)]],
```
Comment (ylvaselling) [inline data/assets/educational/scale/school_bus.asset:45]:
```suggestion
      (10.7 meters).]],
```

File: data/assets/educational/scale/scientist.asset [hunk 7]
```diff
@@ -41,6 +41,7 @@ local ScaleModel = {
   },
   GUI = {
     Name = "Scientist",
+    Description = [[The average height of a human is a bit under 170 cm (5'7")]],
```
Comment (ylvaselling) [inline data/assets/educational/scale/scientist.asset:44]:
```suggestion
    Description = [[The average height of a human is a bit under 170 cm (5'7").]],
```

File: data/assets/examples/dashboarditems.asset (hunk 2 above)
Comment (ylvaselling) [inline data/assets/examples/dashboarditems.asset:109]:
```suggestion
    These are dynamic information texts that will be shown over the rendering
```

File: data/assets/examples/discs.asset (hunk 3 above)
Comment (ylvaselling) [inline data/assets/examples/discs.asset:59]:
```suggestion
  Description = [[Examples of different types of rendered discs.]],
```

File: data/assets/examples/globerotation.asset [hunk 8]
```diff
@@ -55,3 +55,15 @@ asset.onDeinitialize(function()
 end)
 
 asset.export(ExampleGlobeRotation)
+
+
+asset.meta = {
+  Name = "Model Example",
+  Version = "1.0",
+  Description = [[An example that demonstrates how to load a 3D model from a geometry
+    file, together with some basic funcitonality. The model is placed on the surface
```
Comment (ylvaselling) [inline data/assets/examples/globerotation.asset:64]:
```suggestion
    file, together with some basic functionality. The model is placed on the surface
```

File: data/assets/examples/globerotation.asset (hunk 4 above)
Comment (ylvaselling) [inline data/assets/examples/globerotation.asset:65]:
```suggestion
    of a planet using a GlobeRotation and GlobeTranslation.]],
```

File: data/assets/examples/grids.asset [hunk 9]
```diff
@@ -19,6 +19,7 @@ local RadialGrid = {
   },
   GUI = {
     Name = "Example Radial Grid",
+    Description = [[A circular 2D grid, with segments based on the radius and angle]],
```
Comment (ylvaselling) [inline data/assets/examples/grids.asset:22]:
```suggestion
    Description = [[A circular 2D grid, with segments based on the radius and angle.]],
```

File: data/assets/examples/grids.asset [hunk 10]
```diff
@@ -43,6 +44,7 @@ local PlanarGrid = {
   },
   GUI = {
     Name = "Example Grid",
+    Description = [[A basic 2D grid, with a given size and number of segments]],
```
Comment (ylvaselling) [inline data/assets/examples/grids.asset:47]:
```suggestion
    Description = [[A basic 2D grid, with a given size and number of segments.]],
```

File: data/assets/examples/joystickproperty.asset [hunk 11]
```diff
@@ -94,3 +94,14 @@ asset.onInitialize(function()
     "Freeze current scale for Earth. Or release it to the trigger again."
   )
 end)
+
+
+asset.meta = {
+  Name = "Joystick example",
+  Version = "1.0",
+  Description = [[Exaple asset that binds a joystick to use for input and navigation.
```
Comment (ylvaselling) [inline data/assets/examples/joystickproperty.asset:102]:
```suggestion
  Description = [[Example asset that binds a joystick to use for input and navigation.
```

File: data/assets/examples/renderableplaneimageonline.asset [hunk 12]
```diff
@@ -13,6 +13,7 @@ local RenderablePlaneImageOnline = {
     URL = "http://data.openspaceproject.com/examples/renderableplaneimageonline.jpg"
   },
   GUI = {
+    Description = "A plane that loads a texture from the internet",
```
Comment (ylvaselling) [inline data/assets/examples/renderableplaneimageonline.asset:16]:
```suggestion
    Description = "A plane that loads a texture from the internet.",
```

File: data/assets/examples/screenspacebrowser.asset [hunk 13]
```diff
@@ -15,3 +15,14 @@ asset.onDeinitialize(function()
 end)
 
 asset.export(Browser)
+
+
+asset.meta = {
+  Name = "ScreenSpaceBrowser Example",
+  Version = "1.0",
+  Description = [[Example of how to load and show a webpage in the rendering. The loaded
+    webpage is shown in screen space]],
```
Comment (ylvaselling) [inline data/assets/examples/screenspacebrowser.asset:24]:
```suggestion
    webpage is shown in screen space.]],
```

File: data/assets/examples/urlsynchronization.asset [hunk 14]
```diff
@@ -31,10 +31,26 @@ asset.syncedResource({
...
   Url = "http://wms.itn.liu.se/Mercury/Messenger_Mosaic/Messenger_Mosaic.wms",
   UseHash = false
 })
+
+
+asset.meta = {
+  Name = "UrlSynchronization",
+  Version = "1.0",
+  Description = [[Example showing how to load resources (any type of data file) from
+    online sources, using the UrlSyncronization resouce type. These can then be used
```
Comment (ylvaselling) [inline data/assets/examples/urlsynchronization.asset:50]:
```suggestion
    online sources, using the UrlSynchronization resource type. These can then be used
```

File: data/assets/examples/video/videoglobe.asset [hunk 15]
```diff
@@ -27,7 +27,7 @@ asset.export("layer", Layer)
 
 
 asset.meta = {
-  Name = "Video Player Test",
+  Name = "Video Player on Globe Example",
   Version = "1.0",
   Description = "An example asset that shows how to include a video on Earth",
```
Comment (ylvaselling) [inline data/assets/examples/video/videoglobe.asset:32]:
```suggestion
  Description = "An example asset that shows how to include a video on Earth.",
```

File: data/assets/examples/video/videoplane.asset [hunk 16]
```diff
@@ -40,7 +40,7 @@ asset.export(Plane)
 
 
 asset.meta = {
-  Name = "Video Plane Test",
+  Name = "Video Plane Example",
   Version = "1.0",
   Description = "An example asset that shows how to include a video on a plane",
```
Comment (ylvaselling) [inline data/assets/examples/video/videoplane.asset:45]:
```suggestion
  Description = "An example asset that shows how to include a video on a plane.",
```

File: data/assets/examples/video/videoscreenspace.asset [hunk 17]
```diff
@@ -22,7 +22,7 @@ asset.export(ScreenSpace)
 
 
 asset.meta = {
-  Name = "Video ScreenSpace",
+  Name = "ScreenSpace Video Example",
   Version = "1.0",
   Description = "An example asset that shows how to include a video in screen space",
```
Comment (ylvaselling) [inline data/assets/examples/video/videoscreenspace.asset:27]:
```suggestion
  Description = "An example asset that shows how to include a video in screen space.",
```

File: data/assets/examples/video/videosphere.asset [hunk 18]
```diff
@@ -30,7 +30,7 @@ asset.export(Sphere)
 
 
 asset.meta = {
-  Name = "Video Player on Sphere",
+  Name = "Video Player on Sphere Example",
   Version = "1.0",
   Description = "An example asset that shows how to include a video on a sphere",
```
Comment (ylvaselling) [inline data/assets/examples/video/videosphere.asset:35]:
```suggestion
  Description = "An example asset that shows how to include a video on a sphere.",
```

File: data/assets/nightsky/altaz.asset [hunk 19]
```diff
@@ -54,7 +54,11 @@ local AltAzGrid = {
   },
   GUI = {
     Name = "Altitude/Azimuth Grid",
-    Path = "/Other/Grids"
+    Description = [[A local Altitude/Azimuth grid centered around your position on a
+    planetary surface. The grid can be toggled, hidden or shown using the accompanying
+    actiions in the actions panel, under "Night Sky". Use these actions to move it to
```
Comment (ylvaselling) [inline data/assets/nightsky/altaz.asset:59]:
```suggestion
    actions in the actions panel, under "Night Sky". Use these actions to move it to
```

File: data/assets/nightsky/altaz.asset [hunk 20]
```diff
@@ -54,7 +54,11 @@ local AltAzGrid = {
...
+    another planet. The default is Earth]],
```
Comment (ylvaselling) [inline data/assets/nightsky/altaz.asset:60]:
```suggestion
    another planet. The default is Earth.]],
```

File: data/assets/nightsky/altaz.asset [hunk 21]
```diff
@@ -120,3 +124,15 @@ asset.export(AltAzGrid)
...
 asset.export("HideAltaz", HideAltaz.Identifier)
 asset.export("ToggleAltaz", ToggleAltaz.Identifier)
+
+
+asset.meta = {
+  Name = "Altitude/Azimuth Grid",
+  Version = "1.0",
+  Description = [[A local Altitude/Azimuth grid centered around your position on a
+    planetary surface. The asset also includes some actions to toggle, hide or
+    show the grid]],
```
Comment (ylvaselling) [inline data/assets/nightsky/altaz.asset:134]:
```suggestion
    show the grid.]],
```

File: data/assets/nightsky/cardinal_directions.asset [hunk 22]
```diff
@@ -60,6 +60,9 @@ local CardinalDirectionSphere = {
   },
   GUI = {
     Name = "Cardinal Directions",
+    Description = [[A textures sphere showing the cardinal directions.
```
Comment (ylvaselling) [inline data/assets/nightsky/cardinal_directions.asset:63]:
```suggestion
    Description = [[A textured sphere showing the cardinal directions.
```

File: data/assets/nightsky/ecliptic_band.asset [hunk 23]
```diff
@@ -40,6 +40,7 @@ local EclipticLine = {
   },
   GUI = {
     Name = "Ecliptic Line",
+    Description = "A line representation of the Ecliptic plane",
```
Comment (ylvaselling) [inline data/assets/nightsky/ecliptic_band.asset:43]:
```suggestion
    Description = "A line representation of the Ecliptic plane.",
```

File: data/assets/nightsky/ecliptic_band.asset [hunk 24]
```diff
@@ -65,6 +66,7 @@ local EclipticBand = {
   },
   GUI = {
     Name = "Ecliptic Band",
+    Description = "A band representation of the Ecliptic plane",
```
Comment (ylvaselling) [inline data/assets/nightsky/ecliptic_band.asset:69]:
```suggestion
    Description = "A band representation of the Ecliptic plane.",
```

File: data/assets/nightsky/equatorial_band.asset [hunk 25]
```diff
@@ -33,6 +33,7 @@ local EquatorialLine = {
   },
   GUI = {
     Name = "Equatorial Line",
+    Description = "A line representation of the Equatorial plane",
```
Comment (ylvaselling) [inline data/assets/nightsky/equatorial_band.asset:36]:
```suggestion
    Description = "A line representation of the Equatorial plane.",
```

File: data/assets/nightsky/galactic_band.asset [hunk 26]
```diff
@@ -22,6 +22,7 @@ local GalacticBand = {
     },
     GUI = {
         Name = "Galactic Equator Line",
+        Description = "A line representation of the Galactic Equator plane",
```
Comment (ylvaselling) [inline data/assets/nightsky/galactic_band.asset:25]:
```suggestion
        Description = "A line representation of the Galactic Equator plane.",
```

File: data/assets/nightsky/planets.asset [hunk 27]
```diff
@@ -30,6 +30,8 @@ local Mercury = {
   Tag = { "nightsky_billboard" },
   GUI = {
     Name = "Night Sky Mercury",
+    Description = [[A night sky version of the planet Mercury, making it visible as
+      a bright object on the sky (textured representation)]],
```
Comment (ylvaselling) [inline data/assets/nightsky/planets.asset:34]:
```suggestion
      a bright object on the sky (textured representation).]],
```

File: data/assets/nightsky/planets.asset [hunk 28]
```diff
@@ -50,6 +52,8 @@ local Venus = {
   Tag = { "nightsky_billboard" },
   GUI = {
     Name = "Night Sky Venus",
+    Description = [[A night sky version of the planet Venus, making it visible as
+      a bright object on the sky (textured representation)]],
```
Comment (ylvaselling) [inline data/assets/nightsky/planets.asset:56]:
```suggestion
      a bright object on the sky (textured representation).]],
```

File: data/assets/nightsky/planets.asset [hunk 29]
```diff
@@ -90,6 +94,8 @@ local Jupiter = {
   Tag = { "nightsky_billboard" },
   GUI = {
     Name = "Night Sky Jupiter",
+    Description = [[A night sky version of the planet Jupiter, making it visible as
+      a bright object on the sky (textured representation)]],
```
Comment (ylvaselling) [inline data/assets/nightsky/planets.asset:98]:
```suggestion
      a bright object on the sky (textured representation).]],
```

File: data/assets/nightsky/planets.asset [hunk 30]
```diff
@@ -110,6 +116,8 @@ local Saturn = {
   Tag = { "nightsky_billboard" },
   GUI = {
     Name = "Night Sky Saturn",
+    Description = [[A night sky version of the planet Saturn, making it visible as
+      a bright object on the sky (textured representation)]],
```
Comment (ylvaselling) [inline data/assets/nightsky/planets.asset:120]:
```suggestion
      a bright object on the sky (textured representation).]],
```

File: data/assets/examples/grids.asset [hunk 31]
```diff
@@ -107,3 +111,13 @@ asset.export(RadialGrid)
 asset.export(PlanarGrid)
 asset.export(SphericalGrid)
 asset.export(BoxGrid)
+
+
+asset.meta = {
+  Name = "Example Grids",
+  Version = "1.0",
+  Description = [[Example of different types of rendered grids.]],
```
Comment (WeirdRubberDuck) [inline data/assets/examples/grids.asset:119]:
```suggestion
  Description = [[Examples of different types of rendered grids.]],
```

**Comment by WeirdRubberDuck** [comment]
Thanks for finding all my typos, @ylvaselling 😁

**Review by alexanderbock — APPROVED** [review]

**Pull request merged**